constants.py

```diff
@@ -184,6 +187,9 @@
 kLimelightMountingOffset = Translation2d(
     (kRobotLength / 2) - (4.125 * kMetersPerInch), 0.0
 )
+kLimelightVerticalOffset = 25.833857 * kMetersPerInch
```

Comment:
Add comment for where this value came from, was it measured from CAD (define which measurement) or from the physical robot (define how to measure/update this value)